touchforms/formplayer/tests/test_experiment.py

```diff
@@ -0,0 +1,54 @@
+from django.test import SimpleTestCase
+from touchforms.formplayer.experiments import formplayer_string_compare, formplayer_compare
+
+
+class ExperimentTest(SimpleTestCase):
+
+    control = {}
+    candidate = {}
+
+    def testExperimentStringCompare(self):
+        # session_id will never be equal, should be ignored
+        assert formplayer_string_compare("123", "234", current_key="session_id")
```

Comment:
in python unit test the standard is to user `self.assertEqual`: https://docs.python.org/2/library/unittest.html